README.md

```diff
@@ -2,63 +2,64 @@
 page_type: sample
 languages:
 - csharp
+- powershell
 products:
+- azure
+- azure-active-directory
 - dotnet
-description: "Add 150 character max description"
-urlFragment: "update-this-to-unique-url-stub"
+- aspnet
+- ms-graph
+description: "Learn how to acquire an access token on a console application."
+urlFragment: "dotnet-desktop-tutorial"

```

Comment:
Actually, thinking more, we might want to use `ms-identity-dotnet-desktop-tutorial`

Reply:
url updated